index.css

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
- [ ] Please use the same font as the Figma template.

![image](https://user-images.githubusercontent.com/92215878/204813446-9eb084a1-ad6a-42d1-9fea-2679f548f146.png)

Reply:
Done! @jasonfelice